src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -145,6 +148,22 @@ export class CellModel extends Disposable implements ICellModel {
 		return this._metadata;
 	}
 
+	public get markdownCursorPosition(): IPosition {
+		return this._markdownCursorPosition;
+	}
+
+	public set markdownCursorPosition(pos: IPosition) {
+		this._markdownCursorPosition = pos;
+	}
+
+	public get richTextCursorPosition(): ICaretPosition {
```

Comment:
No need to have getters/setters if you're just accessing the property directly - just make the properties public.